comfy_cli/command/run/execution.py

```
@@ -0,0 +1,572 @@
+                yield self.make_tasks_table([task])
+
+
+class WorkflowExecution:
+    def __init__(
+        self,
+        workflow,
+        host,
+        port,
+        verbose,
+        progress,
+        local_paths,
```

Pylint (4.0.5)

[warning] Unused argument 'local_paths'

(W0613)